Export with V0.14.0 and --tla-str crashes

Running this command with a parameter worked fine with version 0.13.0

> tk export environments/default k8s/ --tla-str foo=bar --merge --target Deployment/app

With version 0.14.0 and the paths switch 

> tk export k8s/ environments/default --tla-str foo=bar --merge --target Deployment/app

I get an error
```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/grafana/tanka/pkg/jsonnet.Opts.Clone(0xc0003226f0, 0xc000322720, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/private/tmp/tanka-20210204-53513-1tcpv7x/pkg/jsonnet/eval.go:43 +0x1fa
github.com/grafana/tanka/pkg/tanka.parallelLoadEnvironments(0xc00032c5e0, 0x1, 0x1, 0xc0003226f0, 0xc000322720, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/private/tmp/tanka-20210204-53513-1tcpv7x/pkg/tanka/parallel.go:53 +0x253
github.com/grafana/tanka/pkg/tanka.ExportEnvironments(0xc00032c5e0, 0x1, 0x1, 0x7ffeefbff78a, 0x3, 0xc00014bcd0, 0x0, 0x1)
	/private/tmp/tanka-20210204-53513-1tcpv7x/pkg/tanka/export.go:60 +0x1f8
main.exportCmd.func2(0xc0001cd860, 0xc0001b8930, 0x2, 0x7, 0x0, 0x0)
	/private/tmp/tanka-20210204-53513-1tcpv7x/cmd/tk/export.go:101 +0x5de
github.com/go-clix/cli.(*Command).execute(0xc0001cd860, 0xc0001b88c0, 0x7, 0x7, 0xc0001cd860, 0xc0001b88c0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:118 +0x2a3
github.com/go-clix/cli.(*Command).Execute(0xc0001acd20, 0x0, 0x0)
	/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/go-clix/cli@v0.1.1/command.go:74 +0x154
main.main()
	/private/tmp/tanka-20210204-53513-1tcpv7x/cmd/tk/main.go:58 +0x2eb
```

Environment: mac OS Big Sur, Iterm2, brew installed Tanka

--- a/pkg/jsonnet/eval.go
+++ b/pkg/jsonnet/eval.go
@@ -34,7 +34,8 @@ type Opts struct {
 
 // Clone returns a deep copy of Opts
 func (o Opts) Clone() Opts {
-	var extCode, tlaCode InjectedCode
+	extCode, tlaCode := InjectedCode{}, InjectedCode{}
+
 	for k, v := range o.ExtCode {
 		extCode[k] = v
 	}

--- a/pkg/tanka/find.go
+++ b/pkg/tanka/find.go
@@ -11,6 +11,7 @@ import (
 
 // FindOpts are optional arguments for FindEnvs
 type FindOpts struct {
+	JsonnetOpts
 	Selector labels.Selector
 }
 
@@ -20,7 +21,7 @@ type FindOpts struct {
 // are not checked.
 func FindEnvs(path string, opts FindOpts) ([]*v1alpha1.Environment, error) {
 	// find all environments at dir
-	envs, err := find(path)
+	envs, err := find(path, Opts{JsonnetOpts: opts.JsonnetOpts})
 	if err != nil {
 		return nil, err
 	}
@@ -42,9 +43,9 @@ func FindEnvs(path string, opts FindOpts) ([]*v1alpha1.Environment, error) {
 }
 
 // find implements the actual functionality described at 'FindEnvs'
-func find(path string) ([]*v1alpha1.Environment, error) {
+func find(path string, opts Opts) ([]*v1alpha1.Environment, error) {
 	// try if this has envs
-	list, err := List(path, Opts{})
+	list, err := List(path, opts)
 	if len(list) != 0 && err == nil {
 		// it has. don't search deeper
 		return list, nil
@@ -77,7 +78,7 @@ func find(path string) ([]*v1alpha1.Environment, error) {
 		}
 
 		routines++
-		go findShim(filepath.Join(path, fi.Name()), ch)
+		go findShim(filepath.Join(path, fi.Name()), opts, ch)
 	}
 
 	// collect parallel results
@@ -105,7 +106,7 @@ type findOut struct {
 	err  error
 }
 
-func findShim(dir string, ch chan findOut) {
-	envs, err := find(dir)
+func findShim(dir string, opts Opts, ch chan findOut) {
+	envs, err := find(dir, opts)
 	ch <- findOut{envs: envs, err: err}
 }

--- a/pkg/tanka/parallel.go
+++ b/pkg/tanka/parallel.go
@@ -21,7 +21,7 @@ func parallelLoadEnvironments(paths []string, opts parallelOpts) ([]*v1alpha1.En
 	jobsCh := make(chan parallelJob)
 	list := make(map[string]string)
 	for _, path := range paths {
-		envs, err := FindEnvs(path, FindOpts{opts.Selector})
+		envs, err := FindEnvs(path, FindOpts{Selector: opts.Selector, JsonnetOpts: opts.JsonnetOpts})
 		if err != nil {
 			return nil, err
 		}

